### DISCUSSION TITLE
Visualize GEO IP HTTP Logs into World Map (Kibana build in Map)

### DISCUSSION THREAD

**Discussion by Ax386** [Security-Onion-Solutions/securityonion#2881] [Unsupported Versions]
fixed it ... :-) .. works perfect! ...sorry for spamming the q&a´s ... comming from the lower layers .. ;-) 
stay healthy,
Alex

**Comment by mmasino-matc** [comment]
How did you get this to work?

**Comment by Ax386** [comment]
- fetche the logs via management API, Syslog, or H/NIDS 
- create indexes  for log output in your index pattern
- in kibana point the map to the geo information you got from your indexes (logs)
- you can do it with ISO codes or named keywords

dont forget to reindex your patterns in ELK, SO, ...or system you use, after creating them

hope that helps..
